components/Calender/Modal.vue

```diff
@@ -219,7 +219,8 @@ select {
 }
 
 .select {
-  height: 1.8rem;
+  height: 2rem;
+  padding:0;
   background-color: white;
 
 }
```

Comment:
Move `Modal.vue` to new Folder `Modal` and copy its content in `ModalAddNewEvent`